library/core/src/str/pattern.rs

```diff
@@ -79,6 +81,11 @@ use crate::slice::memchr;
 /// assert_eq!("abaaa".find('b'), Some(1));
 /// assert_eq!("abaaa".find('c'), None);
 ///
+/// // &[char; N]
+/// assert_eq!("ab".find(&['b', 'a']), Some(0));
```

Comment:
Could this be made to work with `assert_eq!("ab".find(['b', 'a']), Some(0));` too?  (AKA no `&`.)